cws2/models/phono_system.py

```diff
@@ -0,0 +1,90 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from cws2.models.base import (
+    TransientModel,
+    TransientModelManager,
+    OwnableModel,
+    UUIDModel,
+)
+from cws2.constants import PhonoSystemStatus, PhoneCategory
+
+
+class PhonoSystem(TransientModel, OwnableModel, UUIDModel):
+    name = models.CharField(
+        verbose_name=_("Name"),
+        max_length=64,
+        db_index=True,
+        help_text=_("This is the name of the phono system."),
+    )
+    status = models.CharField(
+        verbose_name=_("Phono system status"),
+        max_length=1,
+        choices=PhonoSystemStatus.CHOICES,
+        default=PhonoSystemStatus.DRAFT,
+        help_text=_("This is the status of the phono system."),
+    )
+    description = models.TextField(
+        verbose_name=_("Summary"),
+        blank=True,
+        help_text=_(
+            "This is a brief summary explaining the context of the phono system."
+        ),
+    )
+
+    objects = TransientModelManager()
+
+    def __str__(self):
+        return self.name
+
+
+class Phone(TransientModel, UUIDModel):
+    glyph = models.CharField(
+        verbose_name=_("Glyph"),
+        max_length=12,
+        help_text=_("This is a glyph that represents the phone."),
+    )
+    phono_system = models.ForeignKey(
+        "PhonoSystem",
+        on_delete=models.CASCADE,
+        db_column="phono_system_id",
```

Comment:
The `db_column` here is not strictly necessary as it's setting it to the default value

The Django default for `db_column` on a ForeignKey field is the name of the field (e.g in this case `phono_system`) + `_id`. So Django would automatically have done `phono_system_id` here anyway.

It's only useful to set this in case such as where we have a ForeignKey on e.g Users in a field called `owned_by`. As this would auto-generate `owned_by_id` which isn't descriptive, so we set it to `owned_by_user_id`

But yeah, here we should be fine with the default db_column name, so the line isn't necessary :)

https://docs.djangoproject.com/en/4.1/ref/models/fields/#database-representation